src/scene.js

```diff
@@ -795,6 +795,20 @@ export default class Scene {
             catch(error => Promise.resolve({ error }));
     }
 
+    // Query layers within visible tiles
+    queryLayers() {
+        let tile_keys = this.tile_manager.getRenderableTiles().map(t => t.key);
+        return WorkerBroker.postMessage(this.workers, 'self.queryLayers', tile_keys).then(results => {
+            // concatenate results from all workers
+            let filteredResults = [].concat.apply([], results);
+            // de-duplicate results
+            filteredResults = filteredResults.filter(function(item, pos) {
```

Comment:
Let's stick to arrow functions for consistency please, e.g.

`.filter((item, pos) => {`